protocol/messenger_handler.go

```diff
@@ -2282,20 +2277,22 @@ func (m *Messenger) handleChatMessage(state *ReceivedMessageState, forceSeen boo
 	if receivedMessage.ContentType == protobuf.ChatMessage_COMMUNITY {
 		m.logger.Debug("Handling community content type")
 
-		communityResponse, err := m.handleWrappedCommunityDescriptionMessage(receivedMessage.GetCommunity(), common.ShardFromProtobuff(receivedMessage.Shard))
+		signer, description, err := communities.UnwrapCommunityDescriptionMessage(receivedMessage.GetCommunity())
 		if err != nil {
 			return err
 		}
 
-		if communityResponse == nil {
-			return nil
+		err = m.handleCommunityDescription(state, signer, description, receivedMessage.GetCommunity())
+		if err != nil {
+			return err
 		}
 
-		community := communityResponse.Community
-		receivedMessage.CommunityID = community.IDString()
-
-		state.Response.AddCommunity(community)
-		state.Response.CommunityChanges = append(state.Response.CommunityChanges, communityResponse.Changes)
+		if len(description.ID) != 0 {
+			receivedMessage.CommunityID = description.ID
```

Comment:
I do not understand how it can fix the issue... will all necessary data be fetched from the mailserver by `receivedMessage.CommunityID` in this case? 
Here it is a new community with a minted owner token, so this community description will be added to the queue, while the user will receive a message without a description and members count

Reply:
From the description: "This enables clients to obtain invitation's community ID even when description processing is queued."

When we receive this message, we display the card in a pending state and listen for async responses. Since we now possess the community ID, we know which community to monitor in the response, ensuring that the card is eventually displayed in its fully rendered state.

Reply:
Thank you, the proposed solution sounded more scared than the implementation 😄